src/aiortc/contrib/media.py

```diff
@@ -79,6 +80,92 @@ async def stop(self):
         self.__tracks = {}
 
 
+class BroadcasterStreamTrack(MediaStreamTrack):
+    def __init__(self, broadcaster, source: MediaStreamTrack) -> None:
+        super().__init__()
+        self.kind = source.kind
+        self._broadcaster = broadcaster
+        self._queue: asyncio.Queue[Optional[Frame]] = asyncio.Queue()
```

Comment:
```suggestion
        if isinstance(source, RemoteStreamTrack):
            self._queue: asyncio.Queue[Optional[Frame]] = asyncio.Queue()
        else:
            self._queue: asyncio.Queue[Optional[Frame]] = asyncio.Queue(maxsize=1)
```

Tranceiver need always to send the newest frame, while receiver wants an internal buffer

Reply:
I don't want to start have hack for specific stream types. Also setting maxsize to 1 won't give you the most recent frame, but the opposite. And subsequent calls to put_nowait will bomb

Reply:
We can later add some flags to `subscribe()` which would allow us to control the queue size maybe?

Reply:
See PR on your branch